src/onnx/parse_dynamicquantizelinear.cpp

```diff
@@ -110,29 +110,32 @@ struct parse_dynamicquantizelinear : op_parser<parse_dynamicquantizelinear>
 
         // 1. Computing y_scale
         // Note: currently, DynamicQuantizeLinear only has uint8 quantization:
-        const auto x_max = std::numeric_limits<uint8_t>::max();
-        const auto x_min = std::numeric_limits<uint8_t>::min();
+        const auto convert_type_max = std::numeric_limits<uint8_t>::max();
+        const auto convert_type_min = std::numeric_limits<uint8_t>::min();
 
         auto q_range =
             info.add_literal(migraphx::literal{migraphx::shape{x_type}, {x_max - x_min}});
 
+        std::vector<size_t> len_vec(x_reshaped->get_shape().lens().size());
+        std::iota(len_vec.begin(), len_vec.end(), 0);
+
         // maximum(0, max(x))
         auto max_x =
-            info.add_instruction(migraphx::make_op("reduce_max", {{"axes", {0}}}), x_reshaped);
+            info.add_instruction(migraphx::make_op("reduce_max", {{"axes", len_vec}}), x_reshaped);
```

Comment:
Could you please check. I think this was supposed to implement maximum (0, max()). And that is now lost. The 0 was where the len_vec has been provided, and it had perhaps a different intent.. Thanks.

Reply:
Agree, @pfultz2. I just don't see the outer max being implemented anywhere (referring to line 122: `// maximum(0, max(x))` )

Reply:
You mean we should be doing `max(0, reduce_max(x))` not just `reduce_max`?

Reply:
Added.